apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -1300,36 +1300,42 @@ int wm_parse_commands(struct options *opt, int argc, char *argv[])
 	} else if (strcmp(argv[2], "sta") == 0) {
 		opt->func = wm_sta_start;
 	} else if (strcmp(argv[2], "join") == 0) {
-		if (argc < 5) {
+		if (argc < 4) {
```

Comment:
could add comment which params is mandatory?

Reply:
In the old version, all the parameters are mandatory.  Now security_mode may not be supplied if user do not know it.

Reply:
Wrong modification here.
For  command "wm_test join [ssid] [password] [security mode]", argc is max 6.
And here, argc < 5, means "wm_test join [ssid] [password]" case and that is right.
And the same to line 1301.

Reply:
Yes, the argc should be not less than 5. Done.